src/LinearCurve.sol

```diff
@@ -125,16 +109,16 @@ contract LinearCurve is BaseCurve {
     /// @inheritdoc BaseCurve
     /// @notice In a linear curve, the base price will always be 1.  Pool ratio adjustments are dealt with in the EthMultiVault itself.
     function currentPrice(uint256 /*totalShares*/ ) public pure override returns (uint256 sharePrice) {
-        return 1;
```

Comment:
3.6 - currentSharePrice and currentSharePriceCurve Return Price With Orders of Magnitude Inflated by generalConfig.decimalPrecision

We addressed this by updating the comments and ensuring that current share price is always represented by a factor of 1e18, for both pro-rata vaults and curve vaults.